TribbleException should be converted to UserException

The following exception was thrown, which looks a lot like a user exception:

```
htsjdk.tribble.TribbleException$MalformedFeatureFile: Unable to parse header with error: /storageNGS/ngs3/projects/other1/KinderKlinik/Exomes/04_GRCh8_GATK4_cohort/GATK4_testrun/cromwell-executions/HaplotypeCallerGvcf_GATK4/3e5a2ae0-5529-4344-b187-9ceac771e1ed/call-MergeGVCFs/execution/pa.hg38.g.vcf.gz, for input source: file:///storageNGS/ngs3/projects/other1/KinderKlinik/Exomes/04_GRCh8_GATK4_cohort/GATK4_testrun/cromwell-executions/HaplotypeCallerGvcf_GATK4/3e5a2ae0-5529-4344-b187-9ceac771e1ed/call-MergeGVCFs/execution/pa.hg38.g.vcf.gz
at htsjdk.tribble.TribbleIndexedFeatureReader.readHeader(TribbleIndexedFeatureReader.java:254)
at htsjdk.tribble.TribbleIndexedFeatureReader.<init>(TribbleIndexedFeatureReader.java:101)
at htsjdk.tribble.TribbleIndexedFeatureReader.<init>(TribbleIndexedFeatureReader.java:126)
at htsjdk.tribble.AbstractFeatureReader.getFeatureReader(AbstractFeatureReader.java:110)
at org.broadinstitute.hellbender.tools.genomicsdb.GenomicsDBImport.getReaderFromPath(GenomicsDBImport.java:620)
at org.broadinstitute.hellbender.tools.genomicsdb.GenomicsDBImport.getHeaderFromPath(GenomicsDBImport.java:355)
at org.broadinstitute.hellbender.tools.genomicsdb.GenomicsDBImport.initializeHeaderAndSampleMappings(GenomicsDBImport.java:341)
at org.broadinstitute.hellbender.tools.genomicsdb.GenomicsDBImport.onStartup(GenomicsDBImport.java:296)
at org.broadinstitute.hellbender.cmdline.CommandLineProgram.runTool(CommandLineProgram.java:134)
at org.broadinstitute.hellbender.cmdline.CommandLineProgram.instanceMainPostParseArgs(CommandLineProgram.java:179)
at org.broadinstitute.hellbender.cmdline.CommandLineProgram.instanceMain(CommandLineProgram.java:198)
at org.broadinstitute.hellbender.Main.runCommandLineProgram(Main.java:152)
at org.broadinstitute.hellbender.Main.mainEntry(Main.java:195)
at org.broadinstitute.hellbender.Main.main(Main.java:275) Caused by: java.nio.file.NoSuchFileException: /storageNGS/ngs3/projects/other1/KinderKlinik/Exomes/04_GRCh8_GATK4_cohort/GATK4_testrun/cromwell-executions/HaplotypeCallerGvcf_GATK4/3e5a2ae0-5529-4344-b187-9ceac771e1ed/call-MergeGVCFs/execution/pa.hg38.g.vcf.gz
at sun.nio.fs.UnixException.translateToIOException(UnixException.java:86)
at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:102)
at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:107)
at sun.nio.fs.UnixFileSystemProvider.newByteChannel(UnixFileSystemProvider.java:214)
at java.nio.file.Files.newByteChannel(Files.java:361)
at java.nio.file.Files.newByteChannel(Files.java:407)
at htsjdk.samtools.seekablestream.SeekablePathStream.<init>(SeekablePathStream.java:41)
at htsjdk.tribble.util.ParsingUtils.openInputStream(ParsingUtils.java:110)
at htsjdk.tribble.TribbleIndexedFeatureReader.readHeader(TribbleIndexedFeatureReader.java:244) ... 13
```

It looks like we intended to catch and convert to user exception in:
`at org.broadinstitute.hellbender.tools.genomicsdb.GenomicsDBImport.getHeaderFromPath(GenomicsDBImport.java:355)` but we weren't expecting a `TribbleException`

```
    private VCFHeader getHeaderFromPath(final Path variantPath) {
        try(final AbstractFeatureReader<VariantContext, LineIterator> reader = getReaderFromPath(variantPath)) {
            return (VCFHeader) reader.getHeader();
        } catch (final IOException e) {
            throw new UserException("Error while reading vcf header from " + variantPath.toUri(), e);
        }
    }
```

--- a/src/main/java/org/broadinstitute/hellbender/tools/genomicsdb/GenomicsDBImport.java
+++ b/src/main/java/org/broadinstitute/hellbender/tools/genomicsdb/GenomicsDBImport.java
@@ -7,6 +7,7 @@ import htsjdk.samtools.util.Locatable;
 import htsjdk.tribble.AbstractFeatureReader;
 import htsjdk.tribble.CloseableTribbleIterator;
 import htsjdk.tribble.FeatureReader;
+import htsjdk.tribble.TribbleException;
 import htsjdk.tribble.readers.LineIterator;
 import htsjdk.variant.variantcontext.VariantContext;
 import htsjdk.variant.vcf.VCFCodec;
@@ -612,7 +613,11 @@ public final class GenomicsDBImport extends GATKTool {
         final String variantURI = variantPath.toAbsolutePath().toUri().toString();
         final Function<SeekableByteChannel, SeekableByteChannel> cloudWrapper = (cloudPrefetchBuffer > 0 ? is -> SeekableByteChannelPrefetcher.addPrefetcher(cloudPrefetchBuffer, is) : Function.identity());
         final Function<SeekableByteChannel, SeekableByteChannel> cloudIndexWrapper = (cloudIndexPrefetchBuffer > 0 ? is -> SeekableByteChannelPrefetcher.addPrefetcher(cloudIndexPrefetchBuffer, is) : Function.identity());
-        return AbstractFeatureReader.getFeatureReader(variantURI, null, new VCFCodec(), true, cloudWrapper, cloudIndexWrapper);
+        try {
+            return AbstractFeatureReader.getFeatureReader(variantURI, null, new VCFCodec(), true, cloudWrapper, cloudIndexWrapper);
+        } catch (final TribbleException e){
+            throw new UserException("Failed to create reader from " + variantURI, e);
+        }
     }
 
     /**

